README.md

```diff
@@ -67,9 +67,16 @@ Earthdata Search uses PostgreSQL in production on AWS RDS. If you don't already
 
     brew install postgresql
 
+Start the PostgreSQL server:
+    
+    # Intel Macs
+    pg_ctl -D /usr/local/var/postgresql\@14/ start
+    # Apple Silicon
+    pg_ctl -D /opt/homebrew/var/postgresql\@14/ start
```

Comment:
does `brew services start postgresql` work for both?

Reply:
It does, thanks for the tip! I've switched the instructions to use that instead.